### DISCUSSION TITLE
How do I display the progress bar?

### DISCUSSION THREAD

**Discussion by gouki777** [anacrolix/torrent#684] [Q&A] [answered]
I don't know how to configure .How do I display the progress bar?
I can't find the relevant documentation
```
import (
	torrentlogger "github.com/anacrolix/log"
	"github.com/anacrolix/torrent"
	"log"
)
func main() {
	config := torrent.NewDefaultClientConfig()
	config.Logger = torrentlogger.Discard
	config.DownloadRateLimiter
	c, _ := torrent.NewClient(config)
	defer c.Close()
	t, _ := c.AddMagnet("magnet:?xt=urn:btih:xxxxxxxxxxxxxxxxxxxxxx")
	<-t.GotInfo()
	info := t.Info()
	log.Print(info.Name)
	t.DownloadAll()
	c.WaitAll()
	log.Print("OK, torrent downloaded")
}
```

**Comment by anacrolix** [comment, marked as answer]
Take a look at https://github.com/anacrolix/torrent/blob/3d49a29288cad7ee9e47a92e26fd01f0c9fae4b5/cmd/torrent/main.go#L37 and https://github.com/anacrolix/confluence/blob/26e41682a5f577e72183b6b463b2c0c6f201f1cd/confluence/handlers.go#L107 for inspiration.
